dist/browser/bootlint.js

```diff
@@ -11343,6 +11394,34 @@ var LocationIndex = _location.LocationIndex;
             }
         });
     });
+    addLinter("W015", function lintNewBootstrap($, reporter) {
+        var FUTURE_VERSION_ERROR = "Detected what appears to be Bootstrap v4 or later. This version of Bootlint only supports Bootstrap v3.";
```

Comment:
Major versions of Bootstrap are not (fully) backward-compatible with each other, hence Bootstrap v4 will require a new version of Bootlint, whose message here will in turn mention "Bootstrap v5 or later". I suppose we could template this message if we want to be über-future-proof.
See also #168.